data/transactions/logic/doc_test.go

```diff
@@ -105,14 +105,29 @@ func TestAllImmediatesDocumented(t *testing.T) {
 	partitiontest.PartitionTest(t)
 
 	for _, op := range OpSpecs {
-		count := len(op.OpDetails.Immediates)
+		count := len(op.Immediates)
 		note := OpImmediateNote(op.Name)
-		if count == 1 && op.OpDetails.Immediates[0].kind >= immBytes {
+		if count == 1 && op.Immediates[0].kind >= immBytes {
```

Comment:
nit: tho this seems not closely related with this PR, but I don't understand the logic here, seems the opcodes with only one immediate and the immediate is `bytes`, `ints`, `bytess` or `labels` are separately treated?

Reply:
Correct.  All of the complicated immediate types are higher that `immBytes`.  They are complicated enough that the unit test doesn't tear apart the documentation and check it, because they are such special cases.  So we just have to get them right, or our docs will be wrong.

Reply:
got it, thanks!